magento/cart2quote/2017-02-01.yaml

```diff
@@ -0,0 +1,13 @@
+
+title:     Remote Code Execution in Qquoteadv/controllers/DownloadController.php
+link:      https://cart2quote.zendesk.com/hc/en-us/articles/115000616303--FIXED-Security-Vulnerability-in-downloadCustomOptionAction
+cve:       ~
+branches:
+    4:
+        time:     2017-02-01 10:45:00
+        versions: ['>=4.1.6', '<5.0'
+    5:
+        time:     2017-02-01 10:45:00
+        versions: ['>=5.0.1', '<5.4.4']
+reference: composer://cart2quote/cart2quote
```

Comment:
if they have 2 package names, you will have to create 2 advisory files btw (one for each of them, even if everything else stays the same)